File: foundation/foundation 3 month/lesson4/homework/ex_15.py
import random
k=int(input("k ni kiriting: "))
arr=list()
ms=list()
j=list()
for x in range(k):
	arr.append(random.randint(10,20))
for x in range(k):
	ms.append(random.randint(1,1))
for x in arr:
	print(x,end=",")
print("")
for x in range(k):
	j.append(arr[x]+ms[x])
print(j)

File: full stack/full stack 1 month/lesson5/homework/ex_15.js
let n = 5
let arr = []
for (let i = 0; i<n; i++)
{
    arr.push(Math.floor(Math.random()*100))
}
console.log(arr)
let ms =[]
for (let i = 0; i<arr.length;i++)
{
    if(i%2==0)
    {
        ms.push(arr[i])
    }
}
arr = arr.reverse()
for (let i = 0; i<arr.length;i++)
{
    if(i%2==1)
    {
        ms.push(arr[i])
    }
}
console.log(ms)